lib/AcqUnitFilter/AcqUnitFilterContainer.js

```diff
@@ -0,0 +1,29 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import { stripesConnect } from '@folio/stripes/core';
+
+import { getAcqUnitsOptions } from '../utils';
+import { acqUnitsManifest } from '../manifests';
+import AcqUnitFilter from './AcqUnitFilter';
+
+const AcqUnitFilterContainer = ({ resources, ...rest }) => {
+  const acqUnitsOptions = useMemo(() => getAcqUnitsOptions(resources.acqUnits?.records), [resources.acqUnits]);
```

Comment:
this kind a wrong. It should 2 lines, first - extract a dep, const acqUnitsRecords = resources.acqUnits?.records; then you can just use that as a dep

Reply:
also I don't see too much sense to make it here, basically it's enough to add useMemo in `AcqUnitFilter`, and shape could stay

Reply:
Reverted `acqUnitsShape` and moved `getAcqUnitsOptions` to `<AcqUnitFilter />`